### DISCUSSION TITLE
Eth-components 3.0.0-beta16 release

### DISCUSSION THREAD

**Discussion by ghostffcode** [scaffold-eth/eth-components#32] [Announcements]
Add support for `useEthComponent` logic to create custom components
Add `components` with antd default components that utilize `useEthComponent` logic

<hr /><em>This discussion was created from the release <a href='https://github.com/scaffold-eth/eth-components/releases/tag/3.0.0-beta16'>Eth-components 3.0.0-beta16 release</a>.</em>

**Comment by ShravanSunder** [comment]
@ghostffcode  We should have a chat because i think this could have been released with v2 as a stable release.  Its also currently not on devlop branch.
The main branches for this repo are
- master (current stable branch) (v2)
- develop (preview/next branch). (v3 beta)
- any prior stable releases under `releases/` e.g. `releases/v1`

if you want your changes to be part of the next you it needs to be merged into develop with a PR.  I've currently working on eth-hooks v4, so there are a lot of changes related to that in `eth-components v3` beta that have been released.  

I think your changes are probably more stable and meant for a minor release of v2 as its adding functionality.  

Essentially, this breaks the release flow, so we'll have to work around that.

**Comment by ShravanSunder** [comment]
As a separate additional points:
1. I think having issues and PRs helps inform other people who are also working on the repo and with coordination.  
2. we should make a ticket to convert the above components you created to typescript.  this helps with documentation and stability with typechecking for libraries
